lib/src/test/java/com/thenewboston/api/bankapi/repository/BankRepositoryTest.kt

```diff
@@ -4,8 +4,8 @@ import com.thenewboston.api.bankapi.datasource.BankDataSource
 import com.thenewboston.common.http.Outcome
 import com.thenewboston.data.dto.bankapi.accountdto.response.Account
 import com.thenewboston.data.dto.bankapi.accountdto.response.AccountList
-import com.thenewboston.data.dto.bankapi.bankdto.response.BankList
-import com.thenewboston.data.dto.bankapi.bankdto.response.BankTrustResponse
+import com.thenewboston.data.dto.bankcapi.bankdto.response.BankList
+import com.thenewboston.data.dto.bankcapi.bankdto.response.Bank
```

Comment:
The typos in the package name (`bankcapi` instead of `bankapi`) are one of the reasons why the build fails. Please try running `gradle :lib:test` or `gradle :lib:build` locally to make sure all tests pass.